peekaboo-image-picker/src/iosMain/kotlin/com/preat/peekaboo/image/picker/ImagePickerLauncher.ios.kt

```diff
@@ -121,8 +121,8 @@ actual fun rememberImagePickerLauncher(
 }
 
 @OptIn(ExperimentalForeignApi::class)
-private fun UIImage.toByteArray(): ByteArray {
-    val jpegData = UIImageJPEGRepresentation(this, 1.0)!!
+private fun UIImage.toByteArray(compressionQuality: Double): ByteArray {
+    val jpegData = UIImageJPEGRepresentation(this, compressionQuality)!!
```

Comment:
As I mentioned in the comment, let's change this point using a validation check for the compression quality. 
```kotlin
private fun UIImage.toByteArray(compressionQuality: Double): ByteArray {
    val validCompressionQuality = compressionQuality.coerceIn(0.0, 1.0)
    val jpegData = UIImageJPEGRepresentation(this, validCompressionQuality)!!
    // ...
}
```

Reply:
I think attaching [FloatRange](https://developer.android.com/reference/kotlin/androidx/annotation/FloatRange) annotation should be good choice for readability

Reply:
Added FloatRange annotation (and annotation lib to toml and project)